Exercise_4/docs/index.md

```diff
@@ -0,0 +1,62 @@
+# Welcome!
+
+## Overview
+
+This exercise is intended to augment [FHIR for Research Exercises 0-3](https://github.com/mitre/fhir-exercises) with a broad view of EHR, terminology systems, and how USCDI and Implementation Guides like US Core enable remote access and interoperability.
+
+## Sections:
+
+### Part 1: EHR Data
+
+**Background:**  Electronic Health Records (EHR) contain important healthcare-related data. Understanding the purpose and structure of data inside the EHR is a necessary foundation for understanding how to access and exchange it.
```

Comment:
```suggestion
**Background:**  Electronic Health Records (EHR) contain important healthcare-related data. Understanding the purpose and structure of data inside the EHR is a necessary foundation for understanding what data is captured, how it is captured, and how it might be queryable through an API.
````